### PR TITLE
Update Flask and Flask-related packages.

### PR DESCRIPTION
**Pull request by qqss88**
## Summary (required)

- Resolves # [_Issue number_]

_Include a summary of proposed changes._

## How to test the changes locally

-

## Impacted areas of the application
List general components of the application that this PR will affect:

those 4 packages have been updated to the newest version.
Flask==1.02
Flask-Cors==3.0.6
Flask-Script==2.0.6
Flask-RESTful==0.3.6

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,10 +3,10 @@ prance[osv]>=0.11 # use apispec[validation] once we no longer fork apispec
 cfenv==0.5.2
 invoke==0.15.0
 psycopg2==2.7.1
-Flask==0.12.4
-Flask-Cors==3.0.2
-Flask-Script==2.0.5
-Flask-RESTful==0.3.5
+Flask==1.02
```
Comment (vrajmohan) [inline requirements.txt]: Did you mean 1.0.2?

**Comment by vrajmohan** [comment]
It would be better to clean up the spelling/case in the commit message.

File: requirements.txt (hunk 1 above)
Comment (qqss88) [reply requirements.txt]: Yes, good catch. Please help make the corresponding change.

**Comment by vrajmohan** [comment]
It should be OK to squash these commits together.

**Pull request merged**